config/config.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/jinzhu/configor"
```

Comment:
Just noticed this. Do we actually have to write it this way? I thought if we used `godep go build` then it sets the GOPATH up correctly for the build, so the import can be just `"github.com/jinzhu/configor"`, as normal. That's why we'd used `godep go build`.